GitHubCore.ps1

```diff
@@ -540,6 +547,13 @@ function Invoke-GHRestMethod
         Set-TelemetryException -Exception $ex -ErrorBucket $errorBucket -Properties $localTelemetryProperties -NoStatus:$NoStatus
         throw $newLineOutput
     }
+    finally
+    {
+        if ($NoStatus)
```

Comment:
@HowardWolosky note in one of the scenarios I only restore the state if $NoStatus is true, because it would have modified only if it initially had $NoStatus=true.

Reply:
You should only need the assignment to `$securityProtocol` before the main `try`, and then the restoration in the main `finally` (at line 554 right now) without needing to check for `$NoStatus`.  If there's a failure during `NoStatus`, we re-`throw` to the main `catch`.  So, both scenarios can be covered with that single assignment and restoration.

Reply:
Thanks for the clarification, fixed!